code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -127,9 +132,37 @@
 					SSblackbox.record_feedback("tally", "handcuffs", 1, type)
 
 			else
-				L.apply_damage(trap_damage, BRUTE)
+				if(istype(L, /mob/living/simple_animal/hostile/bear))
+					L.apply_damage(trap_damage * 2.5, BRUTE)
+				else
+					L.apply_damage(trap_damage, BRUTE)
 	..()
 
+/obj/item/restraints/legcuffs/beartrap/on_found(mob/finder)
+	if(!armed)
+		return FALSE
+	var/mob/living/L = finder
+	armed = FALSE
```

Comment:
lots of copy pasted code here from `crossed` make a generic proc called `spring_trap` or something

Reply:
Should be resolved with [eff2394](https://github.com/ParadiseSS13/Paradise/pull/21941/commits/eff23948f988a95efe39da1b0bbc341ba521e50c)